### PR TITLE
[16.0][MIG] delivery_postlogistics: Migration to 16.0

### PR DESCRIPTION
**Pull request by lehoangan**

### CODE REVIEW

**Review by StephaneMangin — APPROVED** [review]
LGTM  Well spotted about the context propagation ;)

File: delivery_postlogistics/models/stock_package_type.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from odoo import fields, models
 
 
-class ProductPackaging(models.Model):
-    _inherit = "product.packaging"
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
```
Comment (simahawk) [inline delivery_postlogistics/models/stock_package_type.py:8]: shouldn't we have a migration step to migrate current data from packaging to package type?

Comment (lehoangan) [reply delivery_postlogistics/models/stock_package_type.py:8]: `package_carrier_type` is an Odoo native field from Odoo 15, I don't think we need to migrate data

**Review by simahawk — APPROVED** [review]

**Pull request merged**